.github/workflows/report_news/main.py

```diff
@@ -58,6 +62,56 @@ def fetch_tech_griot() -> list:
     return articles
 
 
+def fetch_peef():
+    """
+    Fetching data from peef
+    """
+
+    url = f'{PEEF_SITEMAP_URL}'
+
+    with urlopen(url) as response:
+        body = response.read()
+
+    xml_raw = ET.fromstring(body)
+
+    articles = []
+    for url in xml_raw.findall('{http://www.sitemaps.org/schemas/sitemap/0.9}url')[-MAX_ARTICLES:]:
+        loc = url.find('{http://www.sitemaps.org/schemas/sitemap/0.9}loc')
```

Comment:
Can you secure each iteration with a try/except ?

Reply:
I updated, I think now only HTTP error is remaining.

Reply:
I am not sure why we use this `MAX_ARTICLES=10` limit here, can you elaborate ?
I left a comment on the other [PR](https://github.com/osscameroon/global-github-actions/pull/127#discussion_r1835634882) regarding what I think we should do here,